.npmrc

```diff
@@ -0,0 +1,2 @@
+engine-strict=true
+legacy-peer-deps=true
```

Comment:
This is necessary because otherwise `npm install` would fail with node 20 (we need to upgrade the dependencies in another PR)